ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbarManager.java

```diff
@@ -84,6 +85,13 @@ public void setTitle(ReactToolbar view, @Nullable String title) {
     view.setTitle(title);
   }
 
+  @ReactProp(name = "rtl")
+  public void setLayoutDirection(ReactToolbar view, Boolean flag) {
+    if (flag.equals(true)) {
```

Comment:
This method should handle any type of updates of this property. Currently it only handles update from any value to RTL=true.

You need to consider the case when rtl is set to true and then updated to be false, and also the case when RTL is set to true or false and then property gets deleted as a result of an update. In that case you'd get `null` as a value (and If I understand correctly it should then use locale-specific direction `LayoutDirection.LOCALE`).